osu.Game/Screens/Edit/Timing/MetronomeDisplay.cs

```diff
@@ -0,0 +1,270 @@
+                                },
+                            }
+                        },
+                    }
+                },
+                new Container
+                {
+                    Name = @"Taper adjust",
+                    Masking = true,
+                    Anchor = Anchor.BottomCentre,
+                    Origin = Anchor.BottomCentre,
+                    Size = new Vector2(triangleSize.X, triangleSize.Y - taper),
+                    Children = new Drawable[]
+                    {
+                        new Circle
+                        {
+                            Name = @"Locking wedge",
+                            Anchor = Anchor.TopCentre,
+                            Origin = Anchor.Centre,
+                            Colour = overlayColourProvider.Background1,
+                            Size = new Vector2(8),
+                        }
+                    },
+                },
+                new Circle
+                {
+                    Name = @"Swing connection point",
+                    Y = swing_vertical_offset,
+                    Anchor = Anchor.BottomCentre,
+                    Origin = Anchor.Centre,
+                    Colour = overlayColourProvider.Colour0,
+                    Size = new Vector2(8)
+                },
+                new Container
+                {
+                    Name = @"Lower cover",
+                    Anchor = Anchor.BottomCentre,
+                    Origin = Anchor.BottomCentre,
+                    RelativeSizeAxes = Axes.X,
+                    Masking = true,
+                    Height = lower_cover_height,
+                    Children = new Drawable[]
+                    {
+                        new Triangle
+                        {
+                            Anchor = Anchor.BottomCentre,
+                            Origin = Anchor.BottomCentre,
+                            Size = triangleSize,
+                            Colour = overlayColourProvider.Background2,
+                            EdgeSmoothness = new Vector2(1),
+                            Alpha = 0.8f
+                        },
+                    }
+                },
+                bpmText = new OsuSpriteText
+                {
+                    Name = @"BPM display",
+                    Colour = overlayColourProvider.Content1,
+                    Anchor = Anchor.BottomCentre,
+                    Origin = Anchor.BottomCentre,
+                    Y = -3,
+                },
+            };
+        }
+
+        private double beatLength;
+
+        private TimingControlPoint timingPoint;
+
+        private bool isSwinging;
+
+        private readonly BindableInt interpolatedBpm = new BindableInt();
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            interpolatedBpm.BindValueChanged(bpm => bpmText.Text = bpm.NewValue.ToString(CultureInfo.CurrentCulture));
```

Comment:
This is a bit weird because `CurrentCulture` is specified but that actually doesn't work well with our localisation setup. Almost nothing else ever uses `CurrentCulture` directly - would either use invariant culture or `.ToLocalisableString()` (which will respond to language changes correctly).

Reply:
Probably better using `ToLocalisableString` for anything involving display.